modules/tensor_mechanics/doc/theory/capped_weak_plane.tex

```diff
@@ -396,6 +396,7 @@ \chapter{Smoothing}
 
 
 \chapter{Constraints and assumptions concerning parameters}
```

Comment:
It would be nice if these tex files were moved to the new documentation system sometime.